packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -66,6 +68,10 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
     [],
   );
 
+  const [virtualCursorElements, setVirtualCursorElements] = React.useState([]);
+  const [vcIndex, setVcIndex] = React.useState(0);
+  const [focusedVcElement, setFocusedVcElement] = React.useState(null);
```

Comment:
nit: use a type here
```suggestion
  const [focusedVcElement, setFocusedVcElement] = React.useState<HTMLElement>(null);
```

Reply:
Fixed as well.